com.dosbox_x.DOSBox-X.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: com.dosbox_x.DOSBox-X
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: dosbox-x
+
+finish-args:	# flatpak permissions
+  - --device=all	# needed for gamepads and serial/parallel
+  - --share=ipc	        # needed for X11
+  - --share=network	# needed for networking (NE2000, IPX)
+  - --socket=x11
+  - --nosocket=wayland  # not compatible with wayland at this point
```

Comment:
```suggestion
```

Reply:
As the comment clearly says, wayland does not work at this point.

Reply:
Well --nosocket=wayland is what you have by default.